packages/memprocfs.vm/tools/chocolateyinstall.ps1

```diff
@@ -3,8 +3,41 @@ Import-Module vm.common -Force -DisableNameChecking
 
 $toolName = 'MemProcFS'
 $category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+# Get latest version from GitHub releases with a curl equivalent in powershell
+$latestFileName = 'MemProcFS_files_and_binaries-win_x64-latest.zip'
+$latestVersionUrl = 'https://api.github.com/repos/ufrisk/MemProcFS/releases/latest'
 
-$zipUrl = 'https://github.com/ufrisk/MemProcFS/releases/download/v5.14/MemProcFS_files_and_binaries_v5.14.11-win_x64-20250501.zip'
-$zipSha256 = '40ce4ac604933dd919444a9cee42431b3fe43813fa4155bef17466b7db03d764'
+if (-not (Get-Command Invoke-RestMethod -ErrorAction SilentlyContinue)) {
+    Write-Error "Invoke-RestMethod is not available. Please ensure you are running PowerShell 3.0 or later."
```

Comment:
We require PowerShell >= 5: https://github.com/mandiant/flare-vm?tab=readme-ov-file#requirements